### ISSUE TITLE
Canvas navigation in Index panel <div>s should be <li>s

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2056]
Current structure of the canvas navigation has `<ul>` as a parent element, but `<div>`s as children. Child nodes of `<ul>` should be `<li>`, which can then contain `<div>`s and other elements as needed.

(Rationale:  list elements give a screen reader context. User can hear the number of items in the list and know where they are in relationship to the whole. `<div>`s can look and behave in the same way visually, but don't provide that context.)

**Comment by cbeer** [comment]
Relatedly, should the parent element by an `<ol>`, since the canvases and presumably.. ordered?

**Comment by jvine** [comment]
I... don't think so but I'm having a hard time explaining why. <ul> gives the necessary context without imposing more of an organization than might be implied by the content.
